Podfile

```diff
@@ -20,18 +20,18 @@ target 'WooCommerce' do
 
   # Use the latest bugfix for coretelephony
   #pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :tag => '0.2.4-beta.1'
-  pod 'Automattic-Tracks-iOS', '~> 0.4.4-beta'
+  pod 'Automattic-Tracks-iOS', '~> 0.4.4-beta.2'
 
-  pod 'Gridicons', '~> 0.20-beta'
+  pod 'Gridicons', '~> 0.20-beta.1'
 
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  #pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'task/wc-support-site-url-login'
-  pod 'WordPressAuthenticator', '~> 1.11.0-beta.13'
+  pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'issue/204-remove-showLoginEmail'
+  #pod 'WordPressAuthenticator', '~> 1.11.0-beta.14'
```

Comment:
We will be changing this back to the `beta` version later when https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/207 is merged, right?

Reply:
Correct! Please excuse the oversight. I updated the PR description with the `do not merge until` section.

Reply:
No worries. I wasn't sure. It's been a while since I changed one of the pods. 😄